packages/styling/src/styles/DefaultPalette.ts

```diff
@@ -46,5 +46,9 @@ export const DefaultPalette: IPalette = {
   tealLight: '#00b294',
   greenDark: '#004b1c',
   green: '#107c10',
-  greenLight: '#bad80a'
+  greenLight: '#bad80a',
+  contrastBlackDisabled: '#00ff00',
```

Comment:
Really, we should be using 'greyText' for disabled. This a bigger topic for all fabric components. There's been a thread around exactly this type of coloring for HC, @dzearing or @betrue-final-final was anything decided on this for fabric all up?